lib/utils.js

```diff
@@ -23,7 +23,7 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 var JWT_ISSUER = exports.JWT_ISSUER = 'learnersguild.org';
 
 function idmGraphQLFetch(graphQLParams) {
-  var token = arguments.length <= 1 || arguments[1] === undefined ? null : arguments[1];
+  var token = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : null;
```

Comment:
My guess is that `babel` got updated somewhere along the way. [There's an optional (defaults to `null`) argument `token` in the `src` version of this](https://github.com/LearnersGuild/idm-jwt-auth/blob/master/src/utils.js#L6). This seems to be `babel`'s way of dealing with optional arguments.